codegen/render/funcs.go

```diff
@@ -193,6 +237,89 @@ func fromYAML(str string) map[string]interface{} {
 	return m
 }
 
+func toNode(v interface{}) goyaml.Node {
```

Comment:
@albertlockett you will be able to integrate your comments addition code here.

Reply:
@josh-pritchard this looks great!

I think when we integrate the comment stuff, since we'll want to make it opt-in, I'll end up passing an additional argument to this to control commenting (e.g. is commenting the yaml enabled, what is our line break length is).

So w/ that said, I wonder if we'd want to eventually have `get_operator_custom_values` and `get_operator_values` return map[string]interface{} and then we'll pipe that into get node ?

```diff
- [[- $customValues = mergeNodes $customValues (get_operator_values $operator) (get_operator_custom_values $operator) ]]
+ [[- $customValues = mergeNodes $customValues (toNode (get_operator_values $operator) $config ) (toNode (get_operator_custom_values $operator) $config)  ]]
```

Reply:
yeah that makes sense to me, seriously feel free to adjust anything that makes sense here when you bring in your changes.